### ISSUE TITLE
Carmelo's feedback

### ISSUE THREAD

**Issue by isaacahuvia** [Poverty-Lab/ACS-Map-Dashboard#28]
1.	I got a lot of error messages until I realized that I wasn’t following the instructions for confirming the variable type, etc. I don’t know if it’s worth the trouble, but I wonder if anything can be done from a design perspective to make this less of an issue. If I am the only one who provides this feedback, you should disregard. I figured it out after a couple of minutes.
2.	I could not get some of the housing variables to plot. 
3.	Some of the variables are a bit confusing to me. Perhaps it’s just that I’m unfamiliar with the ACS tables, but for example: the table "Median Age by Geographic Mobility in the Past Year for Current Residence in the United States" has a variable called “Same house 1 year ago.” The map shows counts of individuals within a neighborhood, which I assume are the number of people int hat neighborhood who lived int he same house one year ago. (The numbers are shockingly low.) If I switch to show percent, however, I get a bunch of neighborhoods where the percent is >100%. Meanwhile there are a bunch of error bubbles on the right telling me that I have selected a median, which cannot be aggregated up to the neighborhood level. If the data can’t be mapped by neighborhood for some reason, it might be better to just disable the map?
4.	Similarly, “Means of Transportation to Work by Poverty Status in the Past 12 Months for Workplace Geography” was a bit confusing to me. The list of variables included both ranges of poverty status and different means of transportation to work. I wanted to look at the intersection of this things, but it wasn’t possible.

In short, when it worked (which it did very well for more common population / demographic characteristics), it was very easy to use and I could see it being very helpful to people. For someone who is unfamiliar with ACS tables, it might be frustrating to try to use this tool for some of the more…complex?…tables.
